ansible/roles/datadog/templates/kafka_consumer.yaml.j2

```diff
@@ -0,0 +1,35 @@
+init_config:
+#  Customize the ZooKeeper connection timeout here
+  zk_timeout: 5
+#  Customize the Kafka connection timeout here
+  kafka_timeout: 5
+
+instances:
+  - kafka_connect_str: localhost:9092  # Should always be deployed to broker so localhost is OK
+    zk_connect_str: "{{ groups['zookeeper'][0] }}:{{ zookeeper_client_port }}"
+    zk_prefix: /0.8
+    consumer_groups:
+      'form-processsor':
```

Comment:
agreed that this list would be hard to maintain, though it doesn't have to be up to date. only if we want stats on a certain group. i'm imagining those groups won't change too often. 

good to note on the groups wasn't certain if we were using them correctly